packages/web3/src/constants.ts

```diff
@@ -23,6 +23,7 @@ export const ALPH_TOKEN_ID = ''.padStart(64, '0')
 export const ONE_ALPH = 10n ** 18n
 export const DUST_AMOUNT = 10n ** 15n
 export const ZERO_ADDRESS = 'tgx7VNFoP9DJiFMFgXXtafQZkUvyEdDHT9ryamHJYrjq'
```

Comment:
Since we are going to use tsdoc, let's try to add a deprecate warning for `ZERO_ADDRESS`